roles/skydive/tasks/provision.yml

```diff
@@ -0,0 +1,77 @@
+---
+- name: oc Command
+  set_fact:
+    cluster_command: oc
+  when: cluster=="openshift"
```

Comment:
you shouldn't need to specify a client.   Use kubectl by default and only use `oc` if `when: cluster=="openshift"` is used.